windows/runner/win32_window.cpp

```diff
@@ -31,8 +31,8 @@ void EnableFullDpiSupportIfAvailable(HWND hwnd) {
           GetProcAddress(user32_module, "EnableNonClientDpiScaling"));
   if (enable_non_client_dpi_scaling != nullptr) {
     enable_non_client_dpi_scaling(hwnd);
-    FreeLibrary(user32_module);
   }
+  FreeLibrary(user32_module);
```

Comment:
I'm curious when `user32_module` would be non-null but `enable_non_client_dpi_scaling` would be null. I suppose if somehow we got a hold of a `User32.dll` that somehow didn't implement that function. Probably unlikely, but this is definitely the right change to make.

Reply:
Yup that's right. [`EnableNonClientDpiScaling` was added in Windows 10](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-enablenonclientdpiscaling#requirements), so I believe this affects Windows 7 and 8 users.